html/obsolete/requirements-for-implementations/other-elements-attributes-and-apis/document-color-fg.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<title>document wg color</title>
+<link rel="author" title="akansha2608" href="mailto:akanshasinghal999@gmail.com">
+<link rel="help" href="https://html.spec.whatwg.org/multipage/obsolete.html#dom-document-fgcolor">
+<link rel="help" href="https://html.spec.whatwg.org/multipage/obsolete.html">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<div id="log"></div>
+<script>
+test(function() {
+var body = document.createElement("body");
+document.documentElement.appendChild(body);
```

Comment:
I’m a bit unsure about why you are creating and removing this body. If it’s essential to the test, it would be good to add a comment.